### DISCUSSION TITLE
Long table generating a 'Bad register code (32768)'

### DISCUSSION THREAD

**Discussion by russurquhart1** [lvjr/tabularray#533] [Q&A]
Hi All,
I have a table, that I exported from Word. The table, in Word, covers about 20 pages for this one table. There is 7 columns to the table and about 15 - 20 rows per page. When i tried to generate a pdf for this LaTeX table I got 'Bad register code (32768)'. I was doing some digging and I got the impression that Tabularray has a limit on the number of rows it can generate into a table, is this correct? The output, below, for the table, was generated by an .xsl script exporting a Word document. (This sample, only has the header row and a single row. The entire table is very large, and I probably couldn't get it all in here.)
Suggestion were to use longtable, but I'd rather use tabularray, if possible. Is there a way to create  a long table without generating the Bad Register Code 32768?

Thanks,

Russ

```
\documentclass[11pt,english,a4paper]{article}
\usepackage[]{fontenc}
\usepackage[latin1]{inputenc}
\usepackage[top=2.5344081101059523cm,bottom=2.5344081101059523cm,left=2.5344081101059523cm,right=2.5344081101059523cm]{geometry}
\usepackage[english]{babel}
\usepackage{graphicx}
\usepackage{eurosym}
\usepackage{amsmath}
\usepackage{color}
\usepackage{fancyhdr}
\usepackage{hyperref}


\begin{document}
\begin{tblr}{colspec={Q[1.46cm,l]Q[3.48cm,l]Q[3.64cm,l]Q[2.36cm,l]Q[1.73cm,l]Q[1.73cm,l]Q[4.57cm,l]}, hlines, vlines, measure=vbox}
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col A}}}}
\end{center}
 & 
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col B}}}}
\end{center}
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col C}}}}
\end{center}
 & 
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col D}}}}
\end{center}
 & 
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col E}}}}
\end{center}
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col F}}}}
\end{center}
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col G}}}}
\end{center}
 & 
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col H/}}}}
\end{center}
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col I}}}}
\end{center}
 & 
\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col J}}}}
\end{center}
 & 

\begin{center}
{\sffamily \textcolor[rgb]{0,0,0}{\textbf{{\scriptsize Col K}}}}
\end{center}
 \\


{\sffamily \textcolor[rgb]{0,0,0}{{\scriptsize A28}}}
 & 

{\sffamily \textcolor[rgb]{0,0,0}{{\scriptsize Not Connected}}}
 & 
\begin{flushright}

\end{flushright}
 & 
\begin{flushright}

\end{flushright}
 & 
\begin{flushright}

\end{flushright}
 & 
\begin{flushright}

\end{flushright}
 & 

\begin{flushright}

\end{flushright}
 \\
...
\end{tblr}
\end{document}
```

**Comment by lvjr** [comment]
You will find #149 if you search `bad register code` in Discussions.

**Reply by russurquhart1** [reply]
So, there isn't a solution within tabularray then, right? As that was 3 years ago, are there any plans for a solution/workaround?

**Reply by lvjr** [reply]
You will find #305 if you search `bad register code` in Issues.

Also when your table is long enough, `tabularray` is quite quite slow at present. Use `tabularray` only when the table is several pages long.

**Reply by russurquhart1** [reply]
Thank you for your help and the information.
